sources/packages/api/src/route-controllers/student/student.controller.service.ts

```diff
@@ -122,4 +134,49 @@ export class StudentControllerService {
       validSin: student.sinValidation.isValidSIN,
     };
   }
+
+  /**
+   * API to fetch all the applications that belong to student.
+   * This API will be used by students.
+   * @param pagination options to execute the pagination.
+   * @returns student application list with total count.
+   */
+  async getStudentApplicationSummary(
+    studentId: number,
+    pagination: ApplicationPaginationOptionsAPIInDTO,
+  ): Promise<PaginatedResultsAPIOutDTO<ApplicationSummaryAPIOutDTO>> {
+    const [applications, count] =
+      await this.applicationService.getAllStudentApplications(
+        studentId,
+        pagination,
+      );
+
+    return {
+      results: applications.map((application: Application) => {
```

Comment:
i think `results: applications.map((application: Application) => 
        this.transformToApplicationSummaryDTO(application);
      ),` shoudl also work

Reply:
Changed